src/utils/rest-api.js

```diff
@@ -865,6 +869,45 @@ export function fetchSecurityAnalysisStatus(studyUuid, currentNodeUuid) {
     return backendFetchText(url);
 }
 
+function getSecurityAnalysisUrl() {
+    return PREFIX_SECURITY_ANALYSIS_SERVER_QUERIES + '/v1/';
+}
+
+export function fetchSecurityAnalysisProviders() {
+    console.info('fetch security analysis providers');
+    const url = getSecurityAnalysisUrl() + 'providers';
+    console.debug(url);
+    return backendFetchJson(url);
+}
+
+export function fetchSecurityAnalysisProvider(studyUuid) {
+    console.info('fetch security analysis provider');
+    const url = getStudyUrl(studyUuid) + '/security-analysis/provider';
+    console.debug(url);
+    return backendFetchText(url);
+}
+
+export function updateSecurityAnalysisProvider(studyUuid, newProvider) {
+    console.info('update security analysis provider');
+    const url = getStudyUrl(studyUuid) + '/security-analysis/provider';
+    console.debug(url);
+    return backendFetch(url, {
+        method: 'POST',
+        headers: {
+            Accept: 'application/json',
+            'Content-Type': 'application/json',
+        },
+        body: newProvider,
+    });
+}
+
+export function fetchDefaultSecurityAnalysisProvider() {
+    console.info('fetch default security analysis provider');
+    const url = PREFIX_STUDY_QUERIES + '/v1/security-analysis-default-provider';
```

Comment:
why not calling directly the security-analysis-server to get the default provider ?

Reply:
Following yesterday slack discussions, we have decided that study server will have its own default provider config (independendly of lf/secu/sensi analysis server). Mainly to avoid study server to query lf/secu/sensi analysis server at study creation time to get default provider and in order to avoid coupling with calculation services at this creation time.

Reply:
ok